thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AutotuneConfigBean.java

```diff
@@ -26,7 +27,7 @@
   // The goal of the autotune
   private double goal;
   // The tuned properties configuration
-  private Map<String, String> configuration;
+  private Map<String, String> configuration = new HashMap<>();
```

Comment:
Is this change truly necessary? There was no problem without this default initialization.

Reply:
It is needed in Anomaly Detection Library.